angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.stories.ts

```diff
@@ -261,6 +260,14 @@ export const defaultStory = () => ({
         }
       ]"
     >
+      <div
+        additional-disclaimer-slot
+        class="sprk-o-Stack__item sprk-o-Stack sprk-o-Stack--large"
```

Comment:
This would use the stack components/directives from Spark.

Reply:
ok! I will also do this in react

Reply:
done!